examples/gno.land/r/gh/accounts.gno

```diff
@@ -0,0 +1,52 @@
+package gh
+
+import "errors"
+
+// Account represents a GitHub user account or organization.
+type Account struct {
+	id   string
+	name string
+	kind string
+}
```

Comment:
Hmm. What about having types User, Organization, and `type Handle interface { Name() string; URL() string; assertHandle() }`? Or `Namespace`